keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/KeelApplicationEvent.kt

```diff
@@ -0,0 +1,61 @@
+package com.netflix.spinnaker.keel.events
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import java.time.Clock
+import java.time.Instant
+
+@JsonTypeInfo(
+  use = JsonTypeInfo.Id.NAME,
+  property = "type",
+  include = JsonTypeInfo.As.PROPERTY
+)
+@JsonSubTypes(
+  JsonSubTypes.Type(value = ApplicationActuationPaused::class, name = "ApplicationActuationPaused"),
+  JsonSubTypes.Type(value = ApplicationActuationResumed::class, name = "ApplicationActuationResumed")
+)
+sealed class KeelApplicationEvent : PersistentEvent() {
```

Comment:
Can this be renamed to `ApplicationEvent`? makes it a bit more generic maybe?

Reply:
That's what I named it originally, but then... https://github.com/spinnaker/keel/pull/842#discussion_r387319892 😃